OpenRA.Game/Actor.cs

```diff
@@ -366,6 +375,18 @@ public bool IsTargetableBy(Actor byActor)
 			return false;
 		}
 
+		public IEnumerable<WPos> GetTargetablePositions()
+		{
+			if (staticTargetablePositions != null)
+				return staticTargetablePositions;
+
+			var enabledTarPosTraits = targetablePositions.Where(Exts.IsTraitEnabled);
```

Comment:
Style nit: please use full words instead of trimming off letters: `enabledTargetablePositions` isn't much longer, but is much more readable.